src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
```

Comment:
The use of `String(robot[key])` is redundant since the object values are already strings or can be implicitly converted to strings when used as property keys.